sql/hive/benchmarks/HiveUDFBenchmark-results.txt

```diff
@@ -0,0 +1,7 @@
+OpenJDK 64-Bit Server VM 1.8.0_352-bre_2022_12_13_23_06-b00 on Mac OS X 13.1
```

Comment:
@yaooqinn please add Java 11 & 17 result too.

Reply:
And the benchmark result should be produced by GA

Reply:
May be we can remove the benchmark?

Reply:
+1 to @wangyum 's suggestion, as after this fix, the benchmarking is mostly evaluating hive udf execution

Reply:
fine to remove the benchmark